quinn-udp/src/unix.rs

```diff
@@ -20,24 +20,48 @@ type IpTosTy = libc::c_uchar;
 #[cfg(not(target_os = "freebsd"))]
 type IpTosTy = libc::c_int;
 
+trait StdUdpSocketExt {
```

Comment:
Given that we have only one caller, the extension trait seems heavy-handed. Are you planning some follow-up that will build on this more?

Reply:
There was no thought behind this really, other than being able to call the function with `socket.only_v6()` syntax